NEWS.md

```diff
@@ -1,46 +1,63 @@
+      - Allow `C + dist`, `C - dist`, `dist - C`, and `C*dist` for a constant `C` and distribution `dist`. (#617)
+  * Allow ignoring data from specific nodes with `model.ignoreData(seqData,morphData)` (#628)
   * Extend `type( )` to optionally show the full type-spec (#504).
-  * Add a generic `vectorFlatten` function (#514).
-  * Extend the starting tree simulator to account for origin/root age (#561).
 
 ## Bug fixes
-  * Fix fossil age sampling with an initial tree in `dnBDSTP` (#480).
-  * Fix FBD checkpointing bugs related to `dnBDSTP` (#484) and fossil time moves (#502, #517).
   * Multiple fixes for building, especially on Windows (#490, #491, #493, #494, #495, #496, #498).
-  * Fix `dnMixture` of rate matrices and other `Cloneable` objects (#501).
-  * Fix `treeAssembly` sometimes failing to initialize branch lengths (#509).
-  * Don't crash if a file changes while we are `source( )`-ing it (#510).
-  * Fix segfault with `dnConstrainedTopology` + `dnUniformTimeTree` (#513).
-  * Fix operator summary for MC^3 when moves are tuned (#522).
-  * Fix FBD initialization issues (#537, #561).
-  * Fix type conversion so that it employs deterministic nodes (#545).
-  * Fix MC^3 checkpointing bug (#553).
-  * Make `RealPos` coherent between conversion and construction (#554).
-  * Fix Gelman-Rubin (PSRF) and stationarity stopping rules (#555).
-  * Don't move fossil tips outside their age ranges (#559).
+  * Checkpointing
+      - Fix FBD checkpointing bugs related to `dnBDSTP` (#484) and fossil time moves (#502, #517).
+      - Fix MC^3 checkpointing bug (#553).
+  * Types
+      - Fix type conversion to integer so that it employs deterministic nodes (#545).
+      - Make `RealPos` coherent between conversion and construction (#554).
+      - Make `vectorFlatten` work on more type (#514).
+  * Crash / NaN
+      - Fix `dnMixture` of rate matrices and other `Cloneable` objects (#501).
+      - Fix `treeAssembly` sometimes failing to initialize branch lengths (#509).
+      - Fix crash if a file changes while we are `source( )`-ing it (#510).
+      - Fix segfault with `dnConstrainedTopology` + `dnUniformTimeTree` (#513).
+      - Fix a segfault in Model::getOrderedStochasticNodes (#569).
+  * MCMC: initialization
+      - Fix fossil age sampling with an initial tree in `dnBDSTP` (#480).
+      - Fix FBD initialization issues (#537, #561).
+      - Extend the starting tree simulator to account for origin/root age (#561).
+  * MCMC
+      - Fix recalculation of likelihoods for brownian motion distributions (#596).
+      - Fix recalculation of likelihoods with mvRootNodeTimeSlideUniform (#594).
+      - Fix recalculation of likelihood with treeAssembly (#549).
+      - Fix Gelman-Rubin (PSRF) and stationarity stopping rules (#555).
+      - Don't move fossil tips outside their age ranges (#559).
+      - Fix operator summary for MC^3 when moves are tuned (#522).
+  * Debug: Log reason for -Inf and NaN probabilities.
```

Comment:
Change this to:

* Debug
    - Log reason for -Inf and NaN probabilities ([#592](https://github.com/revbayes/revbayes/pull/592)).
    
Arguably, PRs [#570](https://github.com/revbayes/revbayes/pull/570), [#575](https://github.com/revbayes/revbayes/pull/575), and [#593](https://github.com/revbayes/revbayes/pull/593) should also be listed under this subheading.

Reply:
I was debating that...  sure,  I'll add those.